settings.sample.py

```diff
@@ -98,10 +98,11 @@ def env(key, default, cast=str):
 
 # Uncomment this option to enable adding an issue comment
 # whenever a pull request passes all checks.
-# OK_COMMENT = env('LINTREVIEW_OK_COMMENT',
-#                 ':+1: No lint errors found.')
+# eg: OK_COMMENT = ':+1: No lint errors found.'
+OK_COMMENT = env('LINTREVIEW_OK_COMMENT', '')
```

Comment:
I did consider that, but this concerned me:
https://github.com/markstory/lint-review/blob/master/settings.sample.py#L4

Since it's going to cast it to the string `False`.
I chose `''` because it evaluates as False.

I don't think this is the best solution, but I think these 2 settings should be driven by env-vars for Docker.